[Settings V2] Upated Fancy Zone and Shortcut Guid default values.

laviusmotileng-ms opened the pull request:
<!-- Enter a brief description/summary of your PR here. What does it fix/what does it change/how was it tested (even manually, if necessary)? -->
## Summary of the Pull Request
- Updated Shortcut Guide settings default values.
- Updated FancyZones settings default values.
- Updated Shortcut guide tests to check for default values.
- Updated FancyZones tests to check for default values.


<!-- Please review the items on the PR checklist before submitting-->
## PR Checklist
* [x] Applies to #2335, #2747
* [x] CLA signed. If not, go over [here](https://cla.opensource.microsoft.com/microsoft/PowerToys) and sign the CLA
* [x] Tests added/passed
* [] Requires documentation to be updated
* [x] I've discussed this with core contributors already. If not checked, I'm ready to accept this work might be rejected in favor of a different grand plan. Issue number where discussion took place: #2335, #2747

<!-- Provide a more detailed description of the PR, other things fixed or any additional comments/features here -->
## Detailed Description of the Pull Request / Additional comments

<!-- Describe how you validated the behavior. Add automated tests wherever possible, but list manual validation steps taken as well -->
## Validation Steps Performed
- Ran and updated tests

enricogior commented:
Sync with master since we removed one setting and added two more.

enricogior commented on src/core/Microsoft.PowerToys.Settings.UI.Lib/FZConfigProperties.cs:
```diff
@@ -11,19 +12,28 @@ public FZConfigProperties()
         {
             this.FancyzonesShiftDrag = new BoolProperty();
```
This is also true by default.

laviusmotileng-ms replied:
resolved here: https://github.com/microsoft/PowerToys/pull/2786/commits/9c5d5706924cc3a674220819dd35657880723eb5

laviusmotileng-ms commented:
> Sync with master since we removed one setting and added two more.

I added tests for the settings added and removed tests for the configuration removed
